packages/ember-container-query/rollup.config.mjs

```diff
@@ -64,13 +63,5 @@ export default {
 
     // Remove leftover build artifacts when starting a new build.
     addon.clean(),
-
-    // Copy Readme and License into published package
-    copy({
-      targets: [
-        { src: '../README.md', dest: '.' },
-        { src: '../LICENSE.md', dest: '.' },
```

Comment:
The relative paths for `src` have been incorrect (should have been `../../README.md`, etc.).